src/installer/publish/prepare-artifacts.proj

```diff
@@ -7,10 +7,13 @@
     <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
   </PropertyGroup>
 
+  <!-- Create all the checksum files. Exclude anything that ends
+       in .nupkg (symbol and regular packages). -->
   <Target Name="CreateChecksums">
     <ItemGroup>
       <ArtifactsForGeneratingChecksums
         Include="@(UploadToBlobStorageFile)"
+        Excldude="*.nupkg"
```

Comment:
Typo `Excldude`

Unless there's something in a later task that carries out what the "excl dude" tells it to. 🙂